### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#305] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-04 181243](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/f6b04dbc-f821-46d2-ad78-0d2150c11f3a)
--- 
![Screenshot 2023-11-04 181252](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/16a177ec-8736-41e1-8c03-6a651db15d2e)

**Comment by IonutCicio** [comment]
1. Si consideri $(\mathbb{Z}, +)$ dato $h \in\mathbb{Z}$, si ha quindi $\langle h \rangle = \\{ k\cdot h, k \in \mathbb{Z} \\}$, ma $k \cdot h = 0 \iff k = 0 \lor h = 0$ (rispetto alla somma), quindi l'ordine di un qualsiasi $h$ è \infty dato che l'ordine è un numero intero positivo, ma non esiste nessun intero positivo per cui $k \cdot h = 0$, a meno che $h = 0$, caso in cui $o(h) = 1$
2. Si consideri $(\mathbb{Z}_n, +)$, si ha quindi $[1] \in \mathbb{Z}_n$ per cui $\langle [1] \rangle = \\{ k \cdot [1], k \in \mathbb{Z} \\}$. Per determinare $o([1])$ basta trovare $k$ tale che $1 \cdot k \equiv_n 0 \implies k \equiv_n 0 \implies k \equiv_n n \implies o([1]) = n, [1] \in \mathbb{Z}_n$ _(ricordo che 0 è l'elemento neutro rispetto alla somma, non 1)_
3. $o(H_d) = k$ dato che $[0] = [n] = k\cdot d$
4. Si consideri $(\mathbb{Z}_15, +)$, per determinare $o([3])$, dato che $\langle [3] \rangle = \\{ k \cdot [3], k \in \mathbb{Z} \\}$, basta determinare $3k \equiv\_{15} 0 \implies k \equiv\_{15} 5 \implies o([3]) = 5$. Non bisogna trovare tutte le soluzioni dell'equazione, dato che basta trovare $n$ *minimo*.

**Comment by 18xChris** [comment]
Convengo con le soluzioni sopra esposte, aggiungo che nel punto 3 la soluzione è k perché n = kd e quindi sommando k volte [d] abbiamo [n]

**Comment by CiottoloMaggico** [comment]
Analogo @CuriousCI 
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/c61f4297-7017-4d42-9e4e-27fefe66a654)
